### PR TITLE
Add application endpoints

### PR DESCRIPTION
**Pull request by GCorbel**
I added some endpoints that was missing for account activity API.

### CODE REVIEW

File: lib/twitter/rest/account_activity.rb [hunk 1]
```diff
@@ -118,17 +118,65 @@ def check_subscription(env_name_or_webhook_id)
       # @see https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference/aaa-enterprise#delete-account-activity-all-env-name-subscriptions
       # @note Deactivate a subscription, Users events will not be sent to the app
       # @rate_limited Yes
-      # @authentication Requires user context - all consumer and access tokens
+      # @authentication Workes with user context and application context
```
Comment (FabienChaynes) [inline lib/twitter/rest/account_activity.rb]:
```suggestion
      # @authentication Works with user context and application context
```

File: lib/twitter/rest/account_activity.rb [hunk 2]
```diff
@@ -118,17 +118,65 @@ def check_subscription(env_name_or_webhook_id)
...
+        if enterprise_api
+          perform_request(:get, "/1.1/account_activity/webhooks/#{env_name_or_webhook_id}/subscriptions/all/list.json")
+        else
+          perform_request(:get, "/1.1/account_activity/all/#{env_name_or_webhook_id}/subscriptions/list.json")
+        end
+      end
+
+    private
+
+      def deactivate_with_user_token(env_name_or_webhook_id)
```
Comment (FabienChaynes) [inline lib/twitter/rest/account_activity.rb]: These endpoints are [deprecated](https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference). We shouldn't use them.

It was even [supposed to be retired](https://developer.twitter.com/en/docs/changelog/standard-premium) on January 15, 2020.

**Review by FabienChaynes — CHANGES_REQUESTED** [review]

File: lib/twitter/rest/account_activity.rb (hunk 2 above)
Comment (GCorbel) [reply lib/twitter/rest/account_activity.rb]: I propose to add a deprecation warning instead of removing it because it continues to work.

Comment (GCorbel) [reply lib/twitter/rest/account_activity.rb]: I removed deprecated endpoints.

File: lib/twitter/rest/account_activity.rb [hunk 3]
```diff
@@ -118,15 +118,49 @@ def check_subscription(env_name_or_webhook_id)
...
       # @note Deactivate a subscription, Users events will not be sent to the app
       # @rate_limited Yes
-      # @authentication Requires user context - all consumer and access tokens
+      # @authentication Works with user context and application context
       # @raise [Twitter::Error::Unauthorized] Error raised when supplied user credentials are not valid.
       # @return [nil]
       # @param env_name_or_webhook_id [String] Environment Name or Webhook Id
-      def deactivate_subscription(env_name_or_webhook_id)
+      # @param user_id [String] Id of the user to unsubscribe
+      def deactivate_subscription(env_name_or_webhook_id, user_id: nil)
```
Comment (FabienChaynes) [inline lib/twitter/rest/account_activity.rb]:
```suggestion
      def deactivate_subscription(env_name_or_webhook_id, user_id)
```

`user_id` isn't optional anymore.

File: lib/twitter/rest/account_activity.rb [hunk 4]
```diff
@@ -118,15 +118,49 @@ def check_subscription(env_name_or_webhook_id)
       # @see https://developer.twitter.com/en/docs/accounts-and-users/subscribe-account-activity/api-reference/aaa-enterprise#delete-account-activity-all-env-name-subscriptions
       # @note Deactivate a subscription, Users events will not be sent to the app
       # @rate_limited Yes
-      # @authentication Requires user context - all consumer and access tokens
+      # @authentication Works with user context and application context
```
Comment (FabienChaynes) [inline lib/twitter/rest/account_activity.rb]:
```suggestion
      # @authentication Requires application context
```

**Review by FabienChaynes — CHANGES_REQUESTED** [review]

File: lib/twitter/rest/account_activity.rb (hunk 3 above)
Comment (GCorbel) [reply lib/twitter/rest/account_activity.rb]: It's fixed.

**Pull request merged**